tutorials/animation/animation_track_types.rst

```diff
@@ -74,6 +74,13 @@ editor, click the curve icon to the right of the animation track.
 In the editor, keys are represented by filled diamonds and the outlined
 diamonds connected to them by a line control curve's shape.
 
+.. tip::
+
+    For better precision while manually working with curves, you might want to alter
+    the zoom levels of the editor. You can do this with your mouse wheel, or using
+    the track on the bottom right of the editor to zoom in on the time axis, 
+    or with :kbd:`Ctrl + Alt + Mouse wheel` to zoom in the values range of the animation.
```

Comment:
```suggestion
    the zoom levels of the editor. The slider on the bottom right of the editor can be used to
    zoom in and out on the time axis, you can also do that with :kbd:`Ctrl + Shift + Mouse wheel`.
    Using :kbd:`Ctrl + Alt + Mouse wheel` will zoom in and out on the Y axis
```

Phrasing needs to be adjusted to be clearer. Also I've never seen a slider called a track before.

Reply:
Good review. Thanks. Sorry about the slider thing.